File: desktop/src/main/java/io/spacepiano/desktop/DesktopLauncher.java
package io.spacepiano.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import io.spacepiano.Game;

public class DesktopLauncher {
    public static void main(String[] args) {
        createApplication();
    }

    private static LwjglApplication createApplication() {
        return new LwjglApplication(new Game(), getDefaultConfiguration());
    }

    private static LwjglApplicationConfiguration getDefaultConfiguration() {
        LwjglApplicationConfiguration configuration = new LwjglApplicationConfiguration();
        configuration.title = "SpacePiano";
        configuration.width = 640;
        configuration.height = 480;
        return configuration;
    }
}

File: desktop/src/main/kotlin/io/spacepiano/desktop/DesktopLauncher.kt
package io.spacepiano.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import io.spacepiano.Game

object DesktopLauncher {
    @JvmStatic fun main(args: Array<String>) {
        createApplication()
    }

    private fun createApplication(): LwjglApplication {
        return LwjglApplication(Game, defaultConfiguration)
    }

    private val defaultConfiguration: LwjglApplicationConfiguration
        get() {
            val configuration = LwjglApplicationConfiguration()
            configuration.title = "SpacePiano"
            configuration.width = 768
            configuration.height = 768
            return configuration
        }
}